testing/sync-test/src/auth.rs

```diff
@@ -129,13 +144,18 @@ impl TestAccount {
         Self::new(
             email,
             password,
-            FxaConfig::stable_dev(CLIENT_ID, REDIRECT_URI),
+            opts.fxa_stack.to_config(CLIENT_ID, REDIRECT_URI),
+            opts.no_delete_account,
         )
     }
 }
 
 impl Drop for TestAccount {
```

Comment:
The separation between `TestAccount` and `TestClient` weirds me out, but I guess it makes sense?